.github/workflows/updater-check.yaml

```diff
@@ -0,0 +1,61 @@
+name: Start Updater
+on:
+  push:
+    branches:
+      - "yli3/**"
```

Comment:
when we merge this, what will happen? Will we change these?

Reply:
We have this discussion here:
https://github.com/stackrox/stackrox/pull/6423#discussion_r1231515897